cloudnet-modules/cloudnet-database-mongodb/src/main/java/de/dytanic/cloudnet/ext/database/mongodb/MongoDatabaseProvider.java

```diff
@@ -0,0 +1,149 @@
+package de.dytanic.cloudnet.ext.database.mongodb;
+
+import com.mongodb.ConnectionString;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoDatabase;
+import de.dytanic.cloudnet.common.concurrent.ITask;
+import de.dytanic.cloudnet.common.concurrent.ListenableTask;
+import de.dytanic.cloudnet.common.document.gson.JsonDocument;
+import de.dytanic.cloudnet.database.AbstractDatabaseProvider;
+import de.dytanic.cloudnet.driver.CloudNetDriver;
+import de.dytanic.cloudnet.driver.database.Database;
+import de.dytanic.cloudnet.ext.database.mongodb.util.DatabaseConnectionData;
+import lombok.Getter;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+@Getter
```

Comment:
Don't use @\_Getter, lombok is only used for @\_ToString and @\_EqualsAndHashCode